R/rebase.R

```diff
@@ -83,6 +83,9 @@ git_reset <- function(type = c("soft", "hard", "mixed"), ref = "HEAD", repo = ".
 #' @useDynLib gert R_git_cherry_pick
 #' @param commit id of the commit to cherry pick
 git_cherry_pick <- function(commit, repo = '.'){
+  if (!is_full_hash(commit)) {
```

Comment:
I got the idea from

https://github.com/r-lib/gert/blob/d0fc50f033dda9e8f9634d20fe47462cb188f40b/R/submodules.R#L48